ImageServer.py

```diff
@@ -0,0 +1,54 @@
+from URLtoPebblePng import get_pebble_png
+import SimpleHTTPServer
+import SocketServer
+import cgi
+import sys
+import urlparse
+
+PORT = 8000
+
+class ServerHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+
+    def do_GET(self):
+    	# For a get request, just return the Pebble_image.png
+    	# This is taken care of by the simeple HTTP server
+    	# request handler
+        return SimpleHTTPServer.SimpleHTTPRequestHandler.do_GET(self)
+
+    def do_POST(self):
+
+     	# Get the length of the image URL from the HTTP headers
+    	length = int(self.headers['Content-length'])
+
+    	# Storing data in a post data structure.
+    	# Can send multiple segments of data which can be decoded and
+    	# stored in post_data.
+    	post_data = urlparse.parse_qs(self.rfile.read(length).decode('utf-8'))
+
+    	# Image url is stored at the imageurl key
+    	ImageURL = str(post_data["imageurl"])
+        print ImageURL
```

Comment:
Please fix the indentation. Lots of stray spaces.

Never use print in your code. Use the logging framework and an appropriate log level.